<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KMComposerAutoCorrectionWidget</class>
 <widget class="QWidget" name="KMComposerAutoCorrectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>372</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>Si&amp;mple Autocorrection</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="upperCase">
         <property name="whatsThis">
          <string>Detect when a new sentence is started and always ensure that the first character is an uppercase character</string>
         </property>
         <property name="text">
          <string>Convert &amp;first letter of a sentence automatically to uppercase
(e.g. &quot;my house. in this town&quot; to &quot;my house. In this town&quot;)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="upperUpper">
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>All words are checked for the common mistake of holding the shift key down a bit too long. If some words must have two uppercase characters, then those exceptions should be added in the 'Exceptions' tab.</string>
         </property>
         <property name="text">
          <string>Convert &amp;two uppercase characters to one uppercase and one lowercase character
 (e.g. PErfect to Perfect)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ignoreDoubleSpace">
         <property name="whatsThis">
          <string>Make sure that more than one space cannot be typed, as this is a common mistake which is quite hard to find in formatted text.</string>
         </property>
         <property name="text">
          <string>&amp;Suppress double spaces</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoReplaceNumber">
         <property name="whatsThis">
          <string>Most standard fraction notations will be converted when available</string>
         </property>
         <property name="text">
          <string>Re&amp;place 1/2... with Â½...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="capitalizeDaysName">
         <property name="text">
          <string>Capitalize name of days</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoFormatUrl">
         <property name="whatsThis">
          <string>Detect when a URL (Uniform Resource Locator) is typed and provide formatting that matches the way an Internet browser would show a URL.</string>
         </property>
         <property name="text">
          <string>Autoformat &amp;URLs (HTML mode)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoChangeFormat">
         <property name="whatsThis">
          <string>When you use _underline_ or *bold*, the text between the underscores or asterisks will be converted to underlined or bold text.</string>
         </property>
         <property name="text">
          <string>Automatically do &amp;bold and underline formatting (HTML mode)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Custom &amp;Quotes</string>
      </attribute>
      <layout class="QGridLayout" name="_3">
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="typographicDoubleQuotes">
         <property name="text">
          <string>Replace &amp;double quotes with typographical quotes</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QPushButton" name="doubleQuote1">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="doubleQuote2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="_5">
         <item>
          <widget class="QPushButton" name="doubleDefault">
           <property name="text">
            <string>Default</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="typographicSingleQuotes">
         <property name="text">
          <string>Replace &amp;single quotes with typographical quotes</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QPushButton" name="singleQuote1">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="singleQuote2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="_7">
         <item>
          <widget class="QPushButton" name="singleDefault">
           <property name="text">
            <string>Default</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Ad&amp;vanced Autocorrection</string>
      </attribute>
      <layout class="QGridLayout" name="_8">
       <item row="1" column="1" rowspan="2">
        <layout class="QVBoxLayout" name="_11">
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="text">
            <string>&amp;Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="advancedAutocorrection">
         <property name="text">
          <string>&amp;Enable word replacement</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="_10">
         <item>
          <widget class="QLabel" name="findLabel">
           <property name="text">
            <string>&amp;Find</string>
           </property>
           <property name="buddy">
            <cstring>find</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KLineEdit" name="find">
           <property name="trapEnterKeyEvent" stdset="0">
            <bool>true</bool>
           </property>
           <property name="showClearButton" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="replaceLabel">
           <property name="text">
            <string>Re&amp;place</string>
           </property>
           <property name="buddy">
            <cstring>replace</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KLineEdit" name="replace">
           <property name="trapEnterKeyEvent" stdset="0">
            <bool>true</bool>
           </property>
           <property name="showClearButton" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QTreeWidget" name="treeWidget">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>Find</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Replace</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>E&amp;xceptions</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Do not treat as the end of a sentence:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="KLineEdit" name="abbreviation">
           <property name="trapEnterKeyEvent" stdset="0">
            <bool>true</bool>
           </property>
           <property name="showClearButton" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1" rowspan="2">
          <layout class="QVBoxLayout" name="_14">
           <item>
            <widget class="QPushButton" name="add1">
             <property name="text">
              <string>&amp;Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="remove1">
             <property name="text">
              <string>R&amp;emove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QListWidget" name="abbreviationList">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Accept two uppercase letters in:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="KLineEdit" name="twoUpperLetter">
             <property name="trapEnterKeyEvent" stdset="0">
              <bool>true</bool>
             </property>
             <property name="showClearButton" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="twoUpperLetterList">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QVBoxLayout" name="_16">
           <item>
            <widget class="QPushButton" name="add2">
             <property name="text">
              <string>A&amp;dd</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="remove2">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="enabledAutocorrection">
     <property name="text">
      <string>Enabled autocorrection</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
